### DISCUSSION TITLE
Automatic code writting

### DISCUSSION THREAD

**Discussion by nicolasdurans** [Cotalker/documentation#747] [3. Ideas & Feature Requests]
La
Nueva funcionalidad que auto escribe los Code en las propiedades debería aplicarse para todas las funcionalidades del Admin. Ejemplo hoy al crear un survey debes poner por cada field un código a mano

**Comment by acedward** [comment]
Agregado a roadmap

**Reply by acedward** [reply]
Recuerdas algún lugar dónde falta?
